### PR TITLE
Boilerplate ready

### PR DESCRIPTION
**Pull request by Saumya29**

### CODE REVIEW

File: src/components/App.js~HEAD [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
```
Comment (ArfatSalman) [inline src/components/App.js~HEAD]: The filename

File: src/index.js~HEAD [hunk 2]
```diff
@@ -2,4 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
```
Comment (ArfatSalman) [inline src/index.js~HEAD]: The filename

File: src/components/App.test.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
```
Comment (ArfatSalman) [inline src/components/App.test.js]: This is not a test. 

Use `expect`  and enzyme's `shallow` to do something like this - 

```js
it("always renders a div", () => {
  const divs = renderScreen().find("div");
  expect(divs.length).toBeGreaterThan(0);
});
```

File: README.md [hunk 4]
```diff
@@ -33,4 +33,4 @@
 
 ## Team Members
  1. [Saumya Tiwari](https://github.com/Saumya29)
- 2. [Nikhil Arora](https://github.com/iamoperand)
\ No newline at end of file
+ 2. [Nikhil Arora](https://github.com/iamoperand)
```
Comment (anshurathee) [inline README.md]: Also include the steps to setup the boilerplate here (in Readme).

Comment (Saumya29) [reply README.md]: On it.

File: src/components/App.test.js (hunk 3 above)
Comment (Saumya29) [reply src/components/App.test.js]: On it.

**Pull request merged**